src/client/components/cube/CubeSubtitle.tsx

```diff
@@ -19,6 +23,9 @@ const CubeSubtitle: React.FC = () => {
       <ResponsiveDiv md>
         <Text md>({subtitle})</Text>
       </ResponsiveDiv>
+      <ResponsiveDiv md baseVisible={true}>
+        <Text md>({mobileSubtitle} Cube)</Text>
```

Comment:
During testing I added a title attribute with the full subtitle name, but unfortunately the mobile browser doesn't show the title popup unlike on desktop. Oh well.